# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/webidl/moz.build
# Commit: d55b6525d7b2
# Full Hash: d55b6525d7b24d90ac7e1b092e4fef698cc244fe
# Author: Andrea Marchesini <amarchesini@mozilla.com>
# Date: 2024-09-13 04:28:43
# Regressor Bug: 1475599
# File Overlap Count: 1
# Description:
#   Bug 1475599 - part 1 - CookieStore API - WebIDL, r=smaug,edgul
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D215144
# ==============================================================================

diff -r 26f637a20dc1 -r d55b6525d7b2 dom/webidl/moz.build
--- a/dom/webidl/moz.build	Thu Sep 12 20:45:28 2024 +0300
+++ b/dom/webidl/moz.build	Thu Sep 12 16:48:18 2024 +0000
@@ -480,6 +480,8 @@
     "Console.webidl",
     "ConstantSourceNode.webidl",
     "ConvolverNode.webidl",
+    "CookieChangeEvent.webidl",
+    "CookieStore.webidl",
     "CreateOfferRequest.webidl",
     "CredentialManagement.webidl",
     "Crypto.webidl",